ansible/roles/helios_broker_setup/tasks/create_secrets.yml

```diff
@@ -1,8 +1,15 @@
 ---
+  - name: Install/update 'apb tools' package if the '{{ apb_tools_cmd }}' command does NOT exist
+    yum:
```

Comment:
We need to come up with a solution for how we can run this on Mac.

Maybe we do a yum install if running on Fedora/RHEL/CentOS
Do pip install if on anything else

Reply:
Also we should update the comment since you took out the logic for checking if apb tool is present before attempting yum install.